### PR TITLE
feat: expose default export names

### PR DESCRIPTION
**Pull request by crowlKats**

### CODE REVIEW

File: src/function.rs [hunk 1]
```diff
@@ -17,6 +17,8 @@ use serde::{Deserialize, Serialize};
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct FunctionDef {
+  #[serde(skip_serializing_if = "Option::is_none", default)]
+  pub default_name: Option<String>,
```
Comment (dsherret) [inline src/function.rs:21]: Maybe we should just store this as something not default export specific (ex. `def_name`, `original_name`, or just `name`)? For exmaple, when a doc node gets renamed we could copy the original name into this property. Keeping the original name might be useful in the future in named export scenarios for some reason:

```
function myFunction() {}

export { myFunction as myNewFunctionName };
```

Comment (crowlKats) [reply src/function.rs]: renamed to `def_name`

File: src/class.rs [hunk 2]
```diff
@@ -243,6 +243,8 @@ impl Display for ClassMethodDef {
 #[derive(Debug, Serialize, Deserialize, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct ClassDef {
+  #[serde(skip_serializing_if = "Option::is_none", default)]
+  pub def_name: Option<String>,
```
Comment (dsherret) [inline src/class.rs:247]: Maybe we should add a comment about when this is set.

**Review by dsherret — APPROVED** [review]
LGTM, though the html code seems lacking in tests.

**Pull request merged**